src/main/java/com/moneymatters/data/models/User.java

```diff
@@ -28,6 +32,18 @@ public class User {
     private String email;
 
     @Column(name = "user_password", nullable = false)
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
     private String password;
 
-}
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL)
```

Comment:
It would be good not having bidirectional entity mapping to avoid performance issue